.gitignore

```diff
@@ -0,0 +1,4 @@
+test.md
+node_modules
+package-lock.json
+package.json
```

Comment:
Good work adding `test.md` and `node_modules` to the .gitignore file. But the `package-lock.json` and `package.json` are not necessary to add to the .gitignore file.
- [ ] Please remove the package-lock.json and package.json from .gitignore file since these two files are necessary to run the project on a different environment. 💪🏼